src/functionalTest/java/uk/gov/hmcts/reform/wacaseeventhandler/controllers/CaseEventHandlerTestingControllerTest.java

```diff
@@ -240,6 +340,6 @@ private Response putEventToRestEndpoint(String messageId, String s2sToken,
             .header(SERVICE_AUTHORIZATION, s2sToken)
             .body(asJsonString(request))
             .when()
-            .put("/messages/" + messageId + (fromDlq ? "?from_dlq=true" : ""));
+            .put("/messages/" + messageId + (fromDlq ? "?from_dlq=true" : "?from_dlq=false"));
```

Comment:
is it worth also testing what the default behavior would be without query params? e.g will it default to false?